delta_rest_client/delta_rest_client.py

```diff
@@ -243,6 +267,14 @@ def create_order_format(price, size, side, product_id, post_only='false'):
     }
     return order
 
+def create_order_format_post(price, size, side, product_id):
```

Comment:
Why do you want to create different functions for different sides?

Reply:
sth like BUY_SIDE = "BUY" is preferrable over just the string "buy". because every exchange follows different conventions and its a lot of work to remember them all. so I suggest adding

BUY_SIDE = "buy"
SELL_SIDE = "sell"

Reply:
I got your point @benjyz. We will add Enums side in the next update. Thanks for your time and effort. Please do let us know in case you want any more enhancements or new methods in the client.